ocs_ci/deployment/vmware.py

```diff
@@ -1570,11 +1589,22 @@ def clone_openshift_installer():
                 clone_type="normal",
             )
         else:
-            clone_repo(
-                constants.VSPHERE_INSTALLER_REPO,
-                upi_repo_path,
-                f"release-{ocp_version}",
-            )
+            # due to failure domain changes in 4.13, use 4.12 branch till
```

Comment:
Can you please open and link a issue here in comment for this to be reverted this change once we have it fixed?

Reply:
done in latest commit